media-sound/lmms/lmms-1.3.0~20250910.recipe

```diff
@@ -0,0 +1,208 @@
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	devel:libfftw3f$secondaryArchSuffix
+	devel:libfltk$secondaryArchSuffix
+	devel:libfltk_forms$secondaryArchSuffix
+	devel:libfltk_images$secondaryArchSuffix
+	devel:libfluidsynth$secondaryArchSuffix >= 2
+	devel:libgme$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libmxml$secondaryArchSuffix
+	devel:libportaudio$secondaryArchSuffix
+	devel:libportmidi$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libsamplerate$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libxcb$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	list_moreutils
+	xml_parser
+	cmd:cmake
+	cmd:doxygen
+	cmd:dot
+	cmd:fltk_config$commandSuffix
+	cmd:fluid$commandSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+
+BUILD()
+{
+	mkdir -p $sourceDir/src/3rdparty/ringbuffer
+	mkdir -p $sourceDir/plugins/{CarlaBase/carla,FreeBoy/game-music-emu,OpulenZ/adplug,Sid/resid/resid,Xpressive/exprtk,ZynAddSubFx/zynaddsubfx}
+	# 3rdparty
+	cp -rf $sourceDir2/ringbuffer-$srcGitRev_2/* $sourceDir/src/3rdparty/ringbuffer
+	# plugins
+	cp -rf $sourceDir3/Carla-$srcGitRev_3/* $sourceDir/plugins/CarlaBase/carla
+	cp -rf $sourceDir4/game-music-emu-$srcGitRev_4/* $sourceDir/plugins/FreeBoy/game-music-emu
+	cp -rf $sourceDir5/adplug-$srcGitRev_5/* $sourceDir/plugins/OpulenZ/adplug
+	cp -rf $sourceDir6/resid-$srcGitRev_6/* $sourceDir/plugins/Sid/resid/resid
+	cp -rf $sourceDir7/exprtk-$srcGitRev_7/* $sourceDir/plugins/Xpressive/exprtk
+	cp -rf $sourceDir8/zynaddsubfx-$srcGitRev_8/* $sourceDir/plugins/ZynAddSubFx/zynaddsubfx
+	# Ladspa plugins
+	cp -rf $sourceDir9/veal-$srcGitRev_9/* $sourceDir/plugins/LadspaEffect/calf/veal
+	cp -rf $sourceDir10/cmt-$srcGitRev_10/* $sourceDir/plugins/LadspaEffect/cmt/cmt
+	cp -rf $sourceDir11/ladspa-$srcGitRev_11/* $sourceDir/plugins/LadspaEffect/swh/ladspa
+	cp -rf $sourceDir12/tap-plugins-$srcGitRev_12/* $sourceDir/plugins/LadspaEffect/tap/tap-plugins
+
+	rm -rf $sourceDir/.gitmodules
+
+	cmake -B build -S . -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$appsDir/LMMS \
+		-DCMAKE_EXE_LINKER_FLAGS="-lnetwork" \
+		-DWANT_QT5=ON \
+		-DWANT_JACK=OFF \
+		-DWANT_WEAKJACK=OFF \
+		-DWANT_VST=OFF \
+		-Wno-dev
+
+	make -C build $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+
+	mkdir -p $prefix/bin $dataDir/bash-completion/completions
+	cp doc/bash-completion/lmms $dataDir/bash-completion/completions
+
+	mv $appsDir/LMMS/bin/lmms $appsDir/LMMS/bin/LMMS
+	ln -s $appsDir/LMMS/bin/LMMS $prefix/bin/lmms
+
+	addResourcesToBinaries $portDir/additional-files/LMMS.rdef \
```

Comment:
The version name in the .rdef is still wrong. This should be changed into a .rdef.in file with the usual sed replacements.